ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/OnDisk.hs

```diff
@@ -533,4 +535,5 @@ data TraceReplayEvent blk replayTo
     -- The @blockInfo@ parameter corresponds replayed block and the @replayTo@
     -- parameter corresponds to the block at the tip of the ImmutableDB, i.e.,
     -- the last block to replay.
+  | UpdateLedgerDbTraceEvent (UpdateLedgerDbTraceEvent blk)
```

Comment:
Please add a Haddock comment on this one, describing the general meaning of the `UpdateLedgerDbTraceEvent` type/category.